### DISCUSSION TITLE
RSVP Payment Description

### DISCUSSION THREAD

**Discussion by dfunnell1** [liedekef/events-made-easy#185] [Q&A] [answered]
Hi All,
I am having difficulties customising payment descriptions for event bookings. I have the following code:
`function my_eme_rsvp_payment_descr($description,$payment,$gateway_name) {
    if ($gateway_name=="sumup"){
                
        $booking_ids = eme_get_bookings_by_paymentid( $payment['id'] );
        $booking = eme_get_booking( $booking_ids[0] );
        $event = eme_get_event($booking['event_id']);
        $person = eme_get_person($booking['person_id']);
        $description = sprintf("%s %s %s Booking",$person['firstname'],$person['lastname'], $event['event_name']);
    }
    return $description;

}`

The filter is firing as the payment is coming through as "Booking". It doesn't seem to be picking up the names of the person or event.

Any ideas would be very much appreciated.

Many thanks

**Comment by liedekef** [comment, marked as answer]
The function to get the booking ids is:
`eme_get_payment_booking_ids($payment['id']);`

**Reply by dfunnell1** [reply]
Awesome, thank you
